### ISSUE TITLE
Define required fields on the JSON Schema

### ISSUE THREAD

**Issue by vitorespindola** [finos-voice/voice-metadata-standard#18]
We should add required fields on JSON Schema file to determined minimum required fields to be compliant with the standard.

Eg. ```"required": ["fileName", "md5Checksum"] ```

**Comment by leop-tcto** [comment]
Agreed. As we begin defining the schema we will do this and then close this issue at that time.
